iFixit/toolbox
# Style tab focus

This styles the focus state of the Tab component.

## Before (on Chrome):
![image](https://user-images.githubusercontent.com/10774982/36703118-075a2858-1b0f-11e8-9357-1d1ee547980e.png)



## After:
![image](https://user-images.githubusercontent.com/10774982/36703020-979402dc-1b0e-11e8-8710-adbe2ea50b8d.png)


## QA
- Go to https://deploy-preview-29--toolbox.netlify.com/#tab and ensure that the focus state for `Tab` matches the after picture.
   - Note: Use dev tools to focus the element
- Ensure consistency on all browsers (screen size is not a factor in this pull)
